packages/fxa-settings/src/models/Account.ts

```diff
@@ -30,7 +30,7 @@ export interface AttachedClient {
   lastAccessTimeFormatted: string;
   approximateLastAccessTime: number;
   approximateLastAccessTimeFormatted: string;
-  location: Location;
+  location: Partial<DeviceLocation>;
```

Comment:
Since you've already made the DeviceLocation properties optional in the interface change above, you don't need this.